### DISCUSSION TITLE
Source code versioning and automatic statistics caluclation from scores

### DISCUSSION THREAD

**Discussion by ge28yen** [langfuse/langfuse#3412] [Ideas]
### Describe the feature or potential improvement

Hi, I had two ideas that are somewhat related: 
1. I would like to version and connect to specific traces not only my prompts, but the whole code underlying the trace. 
2. As I manually annotate the traces, I would love to see an automatic calculation of statistics of my application. I do it with Bayesian statistics right now, with some assumed prior and observations from my annotations, but accuracy/ F1 Scores would perhaps be the easiest to implement in the beggining.

**Comment by marcklingen** [comment]
Have you tried tagging your traces with the current release? see this page for details: https://langfuse.com/docs/experimentation#releases

> As I manually annotate the traces, I would love to see an automatic calculation of statistics of my application. I do it with Bayesian statistics right now, with some assumed prior and observations from my annotations, but accuracy/ F1 Scores would perhaps be the easiest to implement in the beggining.

Agree! there is another open thread on dataset-run-level scores to add aggregate scores, this is a great idea
